frontend/packages/console-shared/src/components/formik-fields/DynamicFormField.tsx

```diff
@@ -16,20 +19,27 @@ const DynamicFormField: React.FC<DynamicFormFieldProps> = ({
   errors,
   formDescription,
   formContext,
+  fullWidth = false,
```

Comment:
Probably better to just default to full width if there's no description provided.

Reply:
@rohitkrai03 update it.